config/serverless-operator.yaml

```diff
@@ -14,6 +14,8 @@ config:
           pullSpec: registry.access.redhat.com/ubi8/openjdk-21
         - name: JAVA_RUNTIME
           pullSpec: registry.access.redhat.com/ubi8/openjdk-21-runtime
+        - name: MUSTGATHER
+          pullSpec: brew.registry.redhat.io/rh-osbs/openshift-ose-must-gather:latest
```

Comment:
As we probably want to use the same version as in https://github.com/openshift-knative/hack/blob/b3ca9b30c790c15ab6105386155c73a3d140de54/cmd/generate/generate.go#L355, we probably need to add this dynamically with the pattern `brew.registry.redhat.io/rh-osbs/openshift-ose-must-gather:v + metadata.Requirements.OcpVersion.Min`.
You can probably do this here: https://github.com/openshift-knative/hack/blob/b3ca9b30c790c15ab6105386155c73a3d140de54/pkg/prowgen/prowgen_konflux.go#L269

Reply:
Right, yeah